src/shared/components/community/sidebar.tsx

```diff
@@ -91,15 +92,25 @@ export class Sidebar extends Component<SidebarProps, SidebarState> {
   }
 
   sidebar() {
+    const myUSerInfo = UserService.Instance.myUserInfo;
+    const { name, actor_id } = this.props.community_view.community;
     return (
       <div>
         <div className="card border-secondary mb-3">
           <div className="card-body">
             {this.communityTitle()}
             {this.props.editable && this.adminButtons()}
-            {this.subscribe()}
+            {myUSerInfo && this.subscribe()}
             {this.canPost && this.createPost()}
-            {this.blockCommunity()}
+            {myUSerInfo && this.blockCommunity()}
+            {!myUSerInfo && (
+              <div className="alert alert-info" role="alert">
```

Comment:
Obvi make a key for this in `lemmy-translations/`. There are examples for how to include the variables in there too. Since you don't need html, you can just do:

`"not_logged_in_message":  "bunch of text... {{apubShortCode}}"`